Detectors/TPC/simulation/include/TPCSimulation/Point.h

```diff
@@ -48,8 +48,6 @@ class ElementalHit {
 // a higher order hit class encapsulating
 // a set of elemental hits belonging to the same trackid (and sector)
 // construct used to do less MC truth linking and to save memory
-// this hitcontainer is linkable with FairLinks,
-// and can be stored as element of a TClonesArray into a branch
 class LinkableHitGroup : public o2::BaseHit {
```

Comment:
Is the name still appropriate after removing `FairLinks`? A short notice should be added to the class description if the name is kept.

Reply:
well spotted

Reply:
Comments have been addressed